R/FilterStateDate.R

```diff
@@ -170,16 +208,19 @@ DateFilterState <- R6::R6Class( # nolint
     #' For this class returned call looks like
     #' `<varname> >= <min value> & <varname> <= <max value>` with
     #' optional `is.na(<varname>)`.
+    #' @param dataname name of data set; defaults to `private$dataname`
     #' @return (`call`)
-    get_call = function() {
-      choices <- as.character(self$get_selected())
+    #'
+    get_call = function(dataname) {
+      if (missing(dataname)) dataname <- private$dataname
+      choices <- as.character(self$get_selected(dataname))
```

Comment:
I guess this `dataname` suppose to go to `get_varname_prefixed`
```suggestion
      choices <- as.character(self$get_selected())
```